contrib_bots/bots/github_issues/github_issues.py

```diff
@@ -48,29 +48,38 @@ def usage(self):
             '''
 
     def triage_message(self, message, client):
```

Comment:
I'm wondering if `triage_message` is also repetitive enough we could just make it more general and move it to the bot library, instead of writing it for every bot separately. What do you think?